examples/guestbook-go/README.md

```diff
@@ -88,91 +93,101 @@ Once that's up you can list the pods in the cluster, to verify that the master a
 
 ### Step Six: Create the guestbook service.
 
-Just like the others, you want a service to group your guestbook pods.  The service specification for the guestbook is in `examples/guestbook-go/guestbook-service.json`.  There's a twist this time - because we want it to be externally visible, we set `"type": "LoadBalancer"` for the service.
+Just like the others, you want a service to group your guestbook pods.  The service specification for the guestbook is in [examples/guestbook-go/guestbook-service.json](guestbook-service.json).  There's a twist this time - because we want it to be externally visible, we set `"type": "LoadBalancer"` for the service.
 
 ```shell
 $ kubectl create -f examples/guestbook-go/guestbook-service.json
 
+      An external load-balanced service was created.  On many platforms (e.g. Google Compute Engine),
+      you will also need to explicitly open a Firewall rule for the service port(s) (tcp:3000) to serve traffic.
+
+      See https://github.com/GoogleCloudPlatform/kubernetes/tree/master/docs/services-firewall.md for more details.
+
 $ kubectl get services
-NAME           LABELS                   SELECTOR                 IP(S)          PORT(S)
-guestbook      name=guestbook           name=guestbook           10.0.114.109   3000/TCP
-redis-master   name=redis,role=master   name=redis,role=master   10.0.11.173    6379/TCP
-redis-slave    name=redis,role=slave    name=redis,role=slave    10.0.234.24    6379/TCP
+NAME                    LABELS                                                                                              SELECTOR                        IP(S)          PORT(S)
+guestbook               app=guestbook                                                                                       app=guestbook                   10.0.217.218   3000/TCP
+                                                                                                                                                            146.148.81.8   
+redis-master            app=redis,role=master                                                                               app=redis,role=master           10.0.136.3     6379/TCP
+redis-slave             app=redis,role=slave                                                                                app=redis,role=slave            10.0.21.92     6379/TCP
+...
 ```
-
-To play with the service itself, find the external IP of the load balancer:
+To play with the service itself, find the external IP of the load balancer. This is reported in the IP column for the guestbook services which shows
+an internal IP address 10.0.217.218 and an external IP address 146.148.81.8 (you may need to scroll right in the box
+above to see the IP column and it make take a few moments to show up). Alternatively, the external IP address can be obtained
+by using `kubectl`:
 
 ```shell
 $ kubectl get services guestbook -o template --template='{{(index .status.loadBalancer.ingress 0).ip}}'
```

Comment:
I don't think it's necessary to use the template stuff here, kubectl get service ... will list the external IP and its more readable.